dadab/dadab.js

```diff
@@ -3,77 +3,101 @@ based on:
  - Simple Image Glitching Processing - datamoshing.com | 2016
  - a pixel sorting sketch from Jerome Martinez | 2017
  */
- 
+
 // output needs to be 1066 x 800
- 
+
+var capture;
+var gif;
 var img;
-var imgFileName = "vera";
-var fileType = "jpg";
+var imgFileName = 'vera';
+var fileType = 'jpg';
+
+var START_FRAME = 15;
+var END_FRAME = 115;
```

Comment:
Kept it short for testing purposes. Will need to change and verify this approach works with 2k frames. But, I gotta jet here in a minute.